src/uct/rocm/base/rocm_base.c

```diff
@@ -157,17 +159,19 @@ hsa_status_t uct_rocm_base_get_ptr_info(void *ptr, size_t size,
         return status;
     }
 
-    if (info.type == HSA_EXT_POINTER_TYPE_UNKNOWN)
-        return HSA_STATUS_ERROR;
-
-    *agent = info.agentOwner;
-
+    if (hsa_mem_type)
```

Comment:
code style:
1. use { } for single-statement blocks (in multiple locations)
2. if (has_mem_type != NULL)

Reply:
done